python/test/test_images.py

```diff
@@ -613,6 +613,67 @@ def test_meanWithSingleRegionIndices3D(self):
         assert_equals(5, collected[0][1][0])
         assert_equals(15, collected[1][1][0])
 
+class TestImagesLocalCorr(PySparkTestCase):
```

Comment:
These tests are good, but the conversion from Series to Images is bit indirect. I'd like to add at least versions of these two tests that explicitly construct data with the same content as `Images` objects, and perform the same operation.